src/locales/en.json

```diff
@@ -361,6 +362,16 @@
     "redirect": "Your password has been changed. You are going to be redirected."
   },
   "LockScreenView": {
+    "choose_pin_body": "choose_pin_body",
+    "choose_pin_cancel": "choose_pin_cancel",
+    "choose_pin_label": "choose_pin_label",
+    "choose_pin_next": "choose_pin_next",
+    "choose_pin_title": "choose_pin_title",
+    "confirm_pin_body": "confirm_pin_body",
+    "confirm_pin_cancel": "confirm_pin_cancel",
+    "confirm_pin_error": "confirm_pin_error",
+    "confirm_pin_next": "confirm_pin_next",
+    "confirm_pin_title": "confirm_pin_title",
```

Comment:
Just a reminder that some english strings are not defined yet :)

Reply:
Yes, we'll have the localisations at a later point I believe, I notified about it